tpl_lib/dxf/dxf.tpl

```diff
@@ -149,46 +149,93 @@ function bounds_center(bounds) {
 }
 
 
-function quad_bezier(p, t) {
-  var c = [sqr(1 - t), 2 * (1 - t) * t, sqr(t)];
+function nurbs_interpolate(t, spl) {
+  var degree = spl.degree;
+  var points = spl.ctrlPts;
+  var knots = spl.knots;
+  var weights = []; /* spl.weights; */ for(i=0; i<points.length; i++) { weights.push(1.0); } /* TODO */
```

Comment:
The weights are in the ``DL_ControlPointData`` structure.  The function Reader::addControlPoint()`` should pass ``ctrlPt.w`` to the ``addControlPoint()`` callback.  The weights could then be added to ``std::vector<double> DXF::Spline::weights`` and then later passed via ``DXFModule`` to TPL.